### DISCUSSION TITLE
Focus after Fingering annotation

### DISCUSSION THREAD

**Discussion by ErlingIv** [musescore/MuseScore#25622] [Ideas]
Hi!
A request:
When entering fingering annotations in a score, the "focus" remains on the (fingering) annotation after it's been added. There is not a lot of "usefull" things to do in that situation. Better then to have the focus on the note for which the fingering was added.
In that way one could "move on", using arrow keys" to the next note and add the next fingering annotation, if so wanted. It simplifies the workflow.

**Comment by MarcSabatella** [comment]
Just press Space while editing the fingering to move to the next note.  The useful thing to do with the fingering selected is fine tune itself position, so losing that would be not good.

**Comment by ErlingIv** [comment]
Hmm, in my case space just starts playing the score.

**Comment by MarcSabatella** [comment]
Then you aren't *editing* the fingeringh.  If you add from the palette, you need to first double-click to edit.  If you add from the menu or a shortcut

For further questions about how to use MuseScore Studio, please ask for help on the official support forum at musescore.org.  Then if it turns out based on discussion there that there is no (easy) way to do the thing you are trying to do, you can come here to GitHub and make a proposal.  But it is usually best to ask for help there first to be sure there isn't already a feature that you just don't know about first.

**Comment by ErlingIv** [comment]
Of course. Should have been on forum, but anyway. Confused myself why I posted it here.
Using keyboard and not mouse and fingering palette is the clue.
A glitch is perhaps that on grace notes the space "jumps over" the "mother note". At least in 3.x.
Thanks anyway!

**Comment by MarcSabatella** [comment]
You can use the palette, you just need to double-click to edit the text in order for Space to take on that behavior.
